### ISSUE TITLE
Setup a ARCH dashboard

### ISSUE THREAD

**Issue by aashish24** [UV-CDAT/uvcdat#654]

**Comment by dlonie** [comment]
Do we still need this? I'll need some hardware for this if so.

**Comment by durack1** [comment]
@aashish24 was this replaced by the buildbot testing? If so a candidate for closure?

**Comment by aashish24** [comment]
Not really. We still need a nightly dashboard for Arch. Buildbot is meant for continuous testing.

**Comment by dlonie** [comment]
Reassigning to Jon, since he has the buildbot knowledge. @aashish24 Is this still something we want to happen? Still waiting on hardware.

**Comment by durack1** [comment]
@aashish24 is this a `milestone 3.0` candidate or can it be slipped into `2.4.0`?

**Comment by doutriaux1** [comment]
@aashish24 do we have tihs?

**Comment by aashish24** [comment]
No not yet
